js/components/dateRange.js

```diff
@@ -0,0 +1,280 @@
+Fliplet.FormBuilder.field('dateRange', {
+  name: 'Date range',
+  category: 'Text inputs',
```

Comment:
We should put this field along with the Date and Time fields into a new category called **Date & time**, after the **Multiple options** category.